Source/JavaScriptCore/runtime/TemporalPlainDate.cpp

```diff
@@ -321,6 +303,85 @@ ISO8601::PlainDate TemporalPlainDate::with(JSGlobalObject* globalObject, JSObjec
     RELEASE_AND_RETURN(scope, TemporalCalendar::isoDateFromFields(globalObject, y, m, d, overflow));
 }
 
+ISO8601::Duration TemporalPlainDate::until(JSGlobalObject* globalObject, TemporalPlainDate* other, JSValue optionsValue)
+{
+    VM& vm = globalObject->vm();
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    if (!calendar()->equals(globalObject, other->calendar())) {
+        RETURN_IF_EXCEPTION(scope, { });
```

Comment:
This will cause debug crash since it is not checking exception on `false` case.

Reply:
Oops! I guess this was just wishful thinking on my part.